readme.txt

```diff
@@ -96,6 +96,10 @@ We'd love your help! Here's a few things you can do:
 
 == Changelog ==
 
+= 3.1.11 - TBD =
```

Comment:
I think we just need to swap out this `TBD` and then this is good to go.

Reply:
@zachwills whoops, yep. just a min.